seqr/views/apis/users_api_tests.py

```diff
@@ -19,26 +19,41 @@
 NON_ANVIL_PROJECT_GUID = 'R0002_empty'
 USERNAME = 'test_user_collaborator'
 USER_OPTION_FIELDS = {'displayName', 'firstName', 'lastName', 'username', 'email', 'isAnalyst'}
+ANALYST_USERNAME = 'test_user'
 
 TOS_VERSION = 2.2
 PRIVACY_VERSION = 1.1
 
 class UsersAPITest(object):
     USERNAME = USERNAME
 
-    @mock.patch('seqr.views.apis.users_api.ANALYST_USER_GROUP', 'analysts')
+    @mock.patch('seqr.views.utils.permissions_utils.ANALYST_PROJECT_CATEGORY', 'analyst-projects')
     @mock.patch('seqr.views.utils.permissions_utils.ANALYST_USER_GROUP', 'analysts')
-    def test_get_all_analyst_options(self):
-        get_all_analyst_url = reverse(get_all_analyst_options)
-        self.check_require_login(get_all_analyst_url)
-        response = self.client.get(get_all_analyst_url)
+    @mock.patch('seqr.views.utils.orm_to_json_utils.ANALYST_USER_GROUP')
+    def test_get_project_collaborator_options(self, mock_analyst_group):
```

Comment:
Could it be tricky to mock a same variable with different values?

Reply:
we do this all over the place in our tests and it works fine?